test/utils/import-utils.js

```diff
@@ -0,0 +1,155 @@
+'use strict';
+
+const { getImportAccessPath } = require('../../rules/utils/import-utils');
+
+const RuleTester = require('eslint').RuleTester;
```

Comment:
Why are you using `RuleTester` to test a function in `import-utils`? Can't you just use Mocha tests here?

Reply:
It is possible to use Mocha without RuleTester. But that test would require parsing with espree, adding a parent field to each node, parsing the scope using eslint-scope, and emulating context.getScope(). I thought using RuleTester would be easier and more reliable testing than doing that, what do you think?
Or are you suggesting that I change the eslint rule for testing to run in eslint, and stop hacking the message validation?

Reply:
It's really confusing to be using `RuleTester` for something other than testing a rule. :) If you really need all of the ESLint infrastructure in place, then using the `Linter` object is a much clearer way to test this functionality.